src/main/cms/cms_menu_imu.c

```diff
@@ -438,6 +465,11 @@ static const OSD_Entry cmsx_menuProfileOtherEntries[] = {
 #ifdef USE_THROTTLE_BOOST
     { "THR BOOST",   OME_UINT8,  NULL, &(OSD_UINT8_t)  { &cmsx_throttleBoost,          0,    100,   1  }   , 0 },
 #endif
+#ifdef USE_ITERM_RELAX
+    { "I_RELAX",         OME_TAB,    NULL, &(OSD_TAB_t)     { &cmsx_iterm_relax,        4, osdTableItermRelax       }, 0 },
```

Comment:
Rather than hardcoding the entry count here (4), it would be better to add a count element to the `enum` for the valid values in `flight/pid.h` - like:
```
typedef enum {
    ITERM_RELAX_OFF,
    ITERM_RELAX_RP,
    ITERM_RELAX_RPY,
    ITERM_RELAX_RP_INC,
    ITERM_RELAX_RPY_INC,
    ITERM_RELAX_COUNT,
} itermRelax_e;

typedef enum {
    ITERM_RELAX_GYRO,
    ITERM_RELAX_SETPOINT,
    ITERM_RELAX_TYPE_COUNT,
} itermRelaxType_e;
```

Then you could use `ITERM_RELAX_COUNT - 1`. This has the benefit again of a single point of maintenance. If the type options were modified the `enum` would have to be updated and this would automatically update the CMS menu option.

Same for the `I_RELAX TYPE` menu entry.

Reply:
Replaced the values with _COUNTS - 1